server/models/migrations/next/migrate.sql

```diff
@@ -1 +1,644 @@
+
+
+
+    CALL drop_constraints('staffing_indice');
+    ALTER TABLE `staffing_indice`
+    ADD CONSTRAINT `staffing_indice__employee` FOREIGN KEY (`employee_uuid`) REFERENCES `employee` (`uuid`),
+    ADD CONSTRAINT `staffing_indice__fonction` FOREIGN KEY (`fonction_id`) REFERENCES `fonction` (`id`),
+    ADD CONSTRAINT `staffing_indice__grade` FOREIGN KEY (`grade_uuid`) REFERENCES `grade` (`uuid`);
+
+
+    CALL drop_constraints('staffing_grade_indice');
+    ALTER TABLE `staffing_grade_indice` ADD CONSTRAINT `staffing_grade_indice__grade` FOREIGN KEY (`grade_uuid`) REFERENCES `grade` (`uuid`);
+
+
+    CALL drop_constraints('staffing_function_indice');
+     ALTER TABLE `staffing_function_indice`
+     ADD CONSTRAINT `staffing_function_indice__fct`  FOREIGN KEY (`fonction_id`) REFERENCES `fonction` (`id`);
+
+    CALL drop_constraints('staffing_indice_parameters');
+     ALTER TABLE `staffing_indice_parameters`
+     ADD  CONSTRAINT `staffing_indice_param__payrall_config` FOREIGN KEY (`payroll_configuration_id`) REFERENCES `payroll_configuration` (`id`);
+
+    CALL drop_constraints('survey_form');
+    ALTER TABLE `survey_form` ADD CONSTRAINT `survey_form__data_collector_management` FOREIGN KEY (`data_collector_management_id`) REFERENCES `data_collector_management` (`id`);
+
+    CALL drop_constraints('survey_data');
+    ALTER TABLE `survey_data` 
+    ADD  CONSTRAINT `survey_data__data_collector_management` FOREIGN KEY (`data_collector_management_id`) REFERENCES `data_collector_management` (`id`),
+    ADD  CONSTRAINT `survey_data__user` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`);
+
+    CALL drop_constraints('survey_data_item');
+    ALTER TABLE `survey_data_item` 
+    ADD CONSTRAINT `survey_data_item__survey_form` FOREIGN KEY (`survey_form_id`) REFERENCES `survey_form` (`id`),
+    ADD CONSTRAINT `survey_data_item__survey_data` FOREIGN KEY (`survey_data_uuid`) REFERENCES `survey_data` (`uuid`);
+
+    CALL drop_constraints('survey_data_log');
+    ALTER TABLE `survey_data_log` 
+    ADD CONSTRAINT `survey_data_log__survey_form` FOREIGN KEY (`survey_form_id`) REFERENCES `survey_form` (`id`),
+    ADD CONSTRAINT `survey_data_log__survey_data` FOREIGN KEY (`survey_data_uuid`) REFERENCES `survey_data` (`uuid`);
+
+    CALL drop_constraints('medical_sheet');
+    ALTER TABLE `medical_sheet` 
+    ADD CONSTRAINT `medical_sheet__survey_data` FOREIGN KEY (`survey_data_uuid`) REFERENCES `survey_data` (`uuid`),
+    ADD CONSTRAINT `medical_sheet__patient` FOREIGN KEY (`patient_uuid`) REFERENCES `patient` (`uuid`);
+
+    
+    CALL drop_constraints('configuration_analysis_tools');
+    ALTER TABLE `configuration_analysis_tools` 
+    ADD CONSTRAINT `config_analysis_tools__acc_ref` FOREIGN KEY (`account_reference_id`) REFERENCES `account_reference` (`id`),
+    ADD CONSTRAINT `config_analysis_tools__analysis_tool_type` FOREIGN KEY (`analysis_tool_type_id`) REFERENCES `analysis_tool_type` (`id`);
+
+    SET foreign_key_checks = 1;
```

Comment:
Should SQL files end in a new line?   I could not find an authoritative answer to this, but github is marking it with a red symbol.  Not sure github really knows the answer, but github is suspicious.